tests/test_sign_cmd.py

```diff
@@ -135,17 +130,17 @@ def test_sign_reject_by_user(cmd,navigator,backend,firmware):
         
     try:
         with cmd.simple_sign_tx_finalize() as ex:
-            sleep(2) # Wait for loading screen to disappear.
+            wait_loading_screen(backend,firmware)
             path = Path(str(path)+"_finalize")
             backend.raise_policy = RaisePolicy.RAISE_ALL
             if firmware.device == "nanos":
                 Instructions = NANOS_NAV_INSTRUCTIONS[:]
                 Instructions.insert(-1,NavIns(NavInsID.RIGHT_CLICK))
-                navigator.navigate_and_compare(Path(__file__).parent.resolve(),path,Instructions,screen_change_after_last_instruction = False)
+                navigator.navigate_and_compare(Path(__file__).parent.resolve(),path,Instructions,screen_change_before_first_instruction = False,screen_change_after_last_instruction = False)
```

Comment:
Why screen_change_before_first_instruction ?

Reply:
`screen_change_before_first_instruction = False` because there is the `wait_loading_screen` called before that will make sure the loading screen is gone and the transaction screen is displayed, so no need to wait for a screen change before navigating.